src/components/HomePage/HomePage.types.ts

```diff
@@ -4,8 +4,11 @@ import { NavigateToAction } from 'decentraland-dapps/dist/modules/location/actio
 import { ProjectState } from 'modules/project/reducer'
 import { createProjectFromTemplate, CreateProjectFromTemplateAction } from 'modules/project/actions'
 
-export type Props = {
+export type DefaultProps = {
   projects: ProjectState['data']
+}
+
+export type Props = DefaultProps & {
```

Comment:
i think this should be `Partial<DefaultProps>`, since they are not actually required

Reply:
[Explained here!](https://github.com/decentraland/builder/pull/160#issuecomment-462733310)